playground/frontend/lib/pages/playground/states/playground_state.dart

```diff
@@ -48,8 +51,20 @@ class PlaygroundState with ChangeNotifier {
     _source = source;
   }
 
+  setExamplesTitle(String examplesTitle) {
+    if (examplesTitle.length > 15) {
```

Comment:
move 15 to a const

Reply:
same here, maybe it's better to convert it to a getter and remove _examplesTitle from a state